src/layouts/home-assistant-main.html

```diff
@@ -116,10 +116,12 @@
       var _this = this;
       this.hass.callApi('get', 'themes').then(
           function (themes) {
+            _this.hass.themes = themes;
```

Comment:
This is not allowed. All data changing on the hass object is done in `<home-assistant>`.

Does themes need to be stored on `hass` anyway? Would it make sense to just store it inside a custom element? ([something like this](https://github.com/ruphin/overwebs/blob/master/app/elements/overwebs-player-data/overwebs-player-data.js))